### ISSUE TITLE
IntelliJ 2018.3

### ISSUE THREAD

**Issue by calvinmuller** [long1eu/flutter_i18n#51]
Any ideas, when clicking the ARB icon in the toolbar.

Added the `misc.xml` file to the .idea directory and get result below.
```
kotlin.KotlinNullPointerException
	at eu.long1.flutter.i18n.workers.I18nFileGenerator.generate(I18nFileGenerator.kt:42)
	at eu.long1.flutter.i18n.workers.Initializer$runActivity$$inlined$scheduleAtFixedRate$1$lambda$1.compute(actions.kt:64)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:963)
	at eu.long1.flutter.i18n.workers.Initializer$runActivity$$inlined$scheduleAtFixedRate$1.run(Timer.kt:150)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
```

**Comment by lionnung** [comment]
I also got this error. @@

**Comment by noordawod** [comment]
Just ignore this error, it's because the initial `strings_en.arb` file is missing.

**Comment by noordawod** [comment]
This will be fixed once #98 is merged.

**Comment by noordawod** [comment]
This is now resolved in v1.1.0. Please upgrade through the Plugins page in your IDE.
